shell/exec.go

```diff
@@ -92,12 +92,12 @@ func (s *MenderShell) IsRunning() bool {
 }
 
 func (s *MenderShell) pipeStdout() {
+	raw := make([]byte, 255)
```

Comment:
while you are at it, please define 255 as something.

Reply:
Done